build.gradle

```diff
@@ -28,3 +79,7 @@ subprojects { project ->
     }
   }
 }
+
+tasks.wrapper {
+  distributionType = Wrapper.DistributionType.ALL
```

Comment:
Why? Bin is best. We don't download the sources for all our dependencies by default, they're downloaded on-demand by the IDE. Keeps CI times quick. Same is true here.